pytype/tests/py3/CMakeLists.txt

```diff
@@ -9,6 +9,16 @@ py_test(
     pytype.tests.test_base
 )
 
+py_test(
+  NAME
+    test_annotated
+  SRCS
+    test_annotated.py
+  DEPS
+    pytype.utils
```

Comment:
The pytype.utils dependency appears to be unneeded.

Reply:
Done.